kinesis.go

```diff
@@ -55,6 +55,7 @@ func (a *KinesisAdapter) Stream(logstream chan *router.Message) {
 		}
 
 		if streamName == "" {
+			debugLog("The stream name is empty, couldn't match the template. Skipping the log.\n")
```

Comment:
What do you mean by things are broken? If the template cannot compile we will return the error: https://github.com/remind101/logspout-kinesis/blob/master/kinesis.go#L31